Moonlight macOS/Utility/keepAlive.m

```diff
@@ -0,0 +1,30 @@
+//
+//  keepAlive.m
+//  LittleHelper
+//
+//  Created by Felix Kratz on 31.10.17.
+//  Copyright © 2017 Felix Kratz. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <IOKit/pwr_mgt/IOPMLib.h>
+#import "keepAlive.h"
+
+
+@implementation keepAlive
+
+CFStringRef reasonForActivity= CFSTR("Moonlight keeps the system awake");
+IOPMAssertionID assertionID = 1984201141;
```

Comment:
What's this magic number mean?

Reply:
Actually, nothing at all. I cleaned it up.